RFPs.md

```diff
@@ -0,0 +1,19 @@
+Any submissions to the Sorobanathon are welcome but for those having trouble getting the cranium cauldion bubbling here are some guaranteed winners.
```

Comment:
Change "guaranteed winners" since it might lead submitters to believe they're guaranteed a reward.  Maybe there's a way to continue the bubbling cauldron metaphor?  (note: looks like "cauldron" is misspelled).   "...for those having trouble getting the cranium cauldron bubbling, here's some eye of newt and toe of frog" or "here's some lighter fuel."  Something like that.